frontend/common/src/components/UserProfile/ProfileSections/WorkPreferencesSection.tsx

```diff
@@ -26,45 +26,107 @@ const WorkPreferencesSection: React.FunctionComponent<{
     : null;
 
   const anyCriteriaSelected = !isEmpty(acceptedOperationalArray);
+
+  // all V2 operational requirements
+  const operationalRequirementsSubsetV2 = [
+    OperationalRequirement.OvertimeOccasional,
+    OperationalRequirement.OvertimeRegular,
+    OperationalRequirement.ShiftWork,
+    OperationalRequirement.OnCall,
+    OperationalRequirement.Travel,
+    OperationalRequirement.TransportEquipment,
+    OperationalRequirement.DriversLicense,
+  ];
+
+  // requirements that have not been selected made into an array
+  const unselectedOperationalArray = operationalRequirementsSubsetV2.filter(
+    (requirement) => !acceptedOperationalRequirements?.includes(requirement),
+  );
+
+  // generate list of unaccepted operational requirements
+  const unacceptedOperationalArray = unselectedOperationalArray
+    ? unselectedOperationalArray.map((opRequirement) => (
+        <li data-h2-font-weight="b(700)" key={opRequirement}>
+          {opRequirement
+            ? getOperationalRequirement(opRequirement).defaultMessage
```

Comment:
These should be wrapped in `intl.formatMessage` so that `react-intl` can properly localise them.

```suggestion
            ? intl.formatMessage(getOperationalRequirement(opRequirement))
```